ntoskrnl/ke/bug.c

```diff
@@ -571,6 +589,19 @@ KeBugCheckUnicodeToAnsi(IN PUNICODE_STRING Unicode,
     return Ansi;
 }
 
+
+VOID
+NTAPI
+AppendAdditionalData(IN CHAR* NewData,
+                     IN BOOLEAN DisplayIt)
+{
+    /* Add new info to AdditionalInformation */
+    sprintf(AdditionalInformation, NewData);
```

Comment:
This makes no sense. `AdditionalInformation` is never initialized to anything but a null pointer.